vespalib/src/vespa/vespalib/stllike/hashtable.h

```diff
@@ -102,15 +102,15 @@ class hash_node {
 public:
     using next_t=hashtable_base::next_t;
     enum {npos=-1u, invalid=-2u};
-    hash_node()
+    hash_node() noexcept
         : _next(invalid)
     {}
-    hash_node(const V & node, next_t next=npos)
+    hash_node(const V & node, next_t next=npos) noexcept
```

Comment:
Since this invokes a copy constructor (which may potentially throw), should this be a conditional `noexcept`? E.g. `noexcept(std::is_nothrow_copy_constructible_v<V>)`

Reply:
Fixed